src/dmd/mtype.d

```diff
@@ -992,6 +993,8 @@ extern (C++) abstract class Type : RootObject
         twstring = twchar.immutableOf().arrayOf();
         tdstring = tdchar.immutableOf().arrayOf();
         tvalist = Target.va_listType();
+        tambig = new TypeFunction(null, null, 0, LINKd);
+        tambig.deco = cast(char*)"@".ptr;
```

Comment:
Dedicating a single character to this is bound to cause problems later. Better to be `__ambiguous`

Reply:
I'm not convinced, it should only be used in error diagnostics and so never end up in object code.